src/app/views/communities/view/overview/overview.ts

```diff
@@ -192,20 +193,11 @@ export default class RouteCommunitiesViewOverview extends BaseRouteComponent {
 	}
 
 	get communityBlockReason() {
-		if (!this.community.block_reason) {
+		if (!this.community.user_block) {
 			return '';
 		}
 
-		return this.defaultBlockReasons[this.community.block_reason] || this.community.block_reason;
-	}
-
-	get defaultBlockReasons(): { [reason: string]: string } {
-		return {
-			spam: this.$gettext('Spam'),
-			'off-topic': this.$gettext('Off Topic'),
-			abuse: this.$gettext('Offensive or insulting'),
-			other: this.$gettext('Other'),
-		};
+		return getBlockReason(this.$gettext, this.community.user_block.reason);
```

Comment:
I don't think this will work. Translation extraction is a hell of a meme, lets not screw around with it.